Update 104-accessibility.md (#1002)
kamranahmedse/developer-roadmap: content/roadmaps/100-frontend/content/101-html/104-accessibility.md

<BadgeLink badgeText='Watch' href='https://youtube.com/playlist?list=PLNYkxOF6rcICWx0C9LVWWVqvHlYJyqw7g'>Complete Playlist on Accessibility</BadgeLink>
<BadgeLink badgeText='Read' colorScheme='yellow' href='https://www.w3schools.com/accessibility/index.php'>Accessibility Tutorial
</BadgeLink>

> <BadgeLink badgeText='Read' colorScheme='yellow' href='https://www.smashingmagazine.com/2021/03/complete-guide-accessible-front-end-components/'>An extensive guide on building accessible components with examples (e.g. buttons, carousels, tabs and many more)
> </BadgeLink>

Reviewer: Please add the title of the article instead.

```suggestion
<BadgeLink badgeText='Read' colorScheme='yellow' href='https://www.smashingmagazine.com/2021/03/complete-guide-accessible-front-end-components/'>A Complete Guide To Accessible Front-End Components</BadgeLink>
```
